policy-controller/src/main.rs

```diff
@@ -97,6 +101,12 @@ async fn main() -> Result<()> {
         .build()
         .await?;
 
+    let probe_networks = if let Some(IpNets(networks)) = probe_networks {
+        Some(networks)
+    } else {
+        None
+    };
```

Comment:
tiny unimportant nit: could just be
```suggestion
    let probe_networks = probe_networks.map(|IpNets(networks))| networks);
```